### DISCUSSION TITLE
MSBuild to build both Android & IOS at the same time?

### DISCUSSION THREAD

**Discussion by nick5454** [cake-build/cake#3329] [General]
Is there a way/extension that will allow me to build both iOS and Android projects in the same solution with one MSBuild command?

I'd like to msbuild both at the same time and grab the apk and ipa in one swoop.

**Comment by nick5454** [comment]
So the answer is yes.

**Reply by augustoproiete** [reply]
@nick5454 It would be great if you expand on that, so that others finding this through search can benefit

**Reply by nick5454** [reply]
You can use this to build in parallel https://docs.microsoft.com/en-us/visualstudio/msbuild/building-multiple-projects-in-parallel-with-msbuild?view=vs-2019

Currently in Visual Studio solutions I create a "Store" configuration and in that I target both iOS and Android for the build.

I haven't tested the parallel build yet on Azure Devops. Once I get my Android build finalized I can
